dotCMS/src/test/java/com/dotcms/util/pagination/TemplatePaginatorTest.java

```diff
@@ -0,0 +1,140 @@
+package com.dotcms.util.pagination;
+
+import com.dotcms.rest.api.v1.template.TemplateHelper;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.PermissionAPI;
+import com.dotmarketing.business.RoleAPI;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.templates.business.TemplateAPI;
+import com.dotmarketing.portlets.templates.model.Template;
+import com.dotmarketing.util.PaginatedArrayList;
+import com.dotmarketing.util.UUIDUtil;
+import com.liferay.portal.model.User;
+import org.junit.Test;
+
+import java.util.Date;
+import java.util.Map;
+
+import static com.dotcms.util.CollectionsUtils.map;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * test {@link TemplatePaginator}
+ */
+public class TemplatePaginatorTest {
```

Comment:
Remember to add the new tests to the MainSuite, if not added it will not run.

Reply:
This is not an integration, just unit test